common/lib/xmodule/xmodule/js/spec/video/html5_video_spec.js

```diff
@@ -105,8 +103,9 @@
                         jasmine.waitUntil(function() {
                             return state.videoPlayer.player.getPlayerState() !== STATUS.PAUSED;
                         }).then(function() {
-                            expect(state.videoPlayer.player.getPlayerState())
-                                .toBe(STATUS.BUFFERING);
+                            expect([STATUS.BUFFERING, STATUS.PLAYING]).toContain(
```

Comment:
@muhammad-ammar can you also check the same condition being used in other tests ? if yes, we probably need to fix those tests as well.